eng/tools/dependency-testing/index.js

```diff
@@ -208,8 +208,9 @@ async function insertTsConfigJson(targetPackagePath, testFolder) {
 async function readAndReplaceSourceReferences(filePath, packageName) {
   var fileContent = await packageUtils.readFile(filePath);
   console.log("Reading filePath = " + filePath);
+  textAssetsContent = fileContent.replace('path.resolve(path.join(process.cwd(), "test-assets"))','path.resolve(path.join(process.cwd(),"..","..", "test-assets"))');
```

Comment:
Typo: should be `testAssetsContent` or simply just keep the name `fileContent`, since we're not changing the asset contents.

--

I'm concerned about the fragility of this approach. This will only work for the specific way that we currently reference static test assets in Form Recognizer, so with that in mind it's an FR-specific workaround. If I were to write some code in the future like:

```
path.join(process.cwd(), "test-assets", "path", "to", "specific", "file.pdf")
```

Then that would break. This workaround would add a layer of magic that locks us into always using `ASSET_PATH` as it's declared.
```

CC @jeremymeng @xirzec is using static test-assets (static files like PDF, JPEG, etc. that we send to the service) something that we do in other packages? If so we may want a more architecturally rigid solution, but otherwise I'm okay with doing a workaround until Form Recognizer has a better story for its static assets.

Reply:
@willmtemple Do you think something like this - `fileContent.replace('path.resolve(path.join(process.cwd(), "test-assets"','path.resolve(path.join(process.cwd(),"..","..", "test-assets"');` where I am only adding a "..",".." (implying the two folder levels for "test/public") will be fragile too? 

So in this example - `path.join(process.cwd(), "test-assets", "path", "to", "specific", "file.pdf")`  with this code it will replace it like this - `path.join(process.cwd(), "..",",..","test-assets", "path", "to", "specific", "file.pdf")`

Reply:
> s using static test-assets (static files like PDF, JPEG, etc. that we send to the service) something that we do in other packages?

FR is the only lib I know which uses static test assets. We could've stored them in blob storage but then would need to download them to local first before running the scenarios.

Reply:
Updated the test assets path to the following to include the scenarios with example pointed above - 
`testAssetsContent = fileContent.replace('path.resolve(path.join(process.cwd(), "test-assets"','path.resolve(path.join(process.cwd(),"..","..", "test-assets"');`

cc: @willmtemple